src/custom/ShareModal/style.tsx

```diff
@@ -0,0 +1,70 @@
+import { styled } from '@mui/material';
+import { DialogContentText, FormControl, ListItemText, Select, Typography } from '../../base';
+
+export const CustomText = styled(Typography)(() => ({
```

Comment:
why do we need to redefine these styles for typography.

Reply:
we don't have to . I will revert that because we have it set from globally.